aiohttp/web_urldispatcher.py

```diff
@@ -169,7 +227,7 @@ def url(self, *, filename, query=None):
         return self._append_query(url, query)
 
     @asyncio.coroutine
-    def handle(self, request):
+    def handler(self, request):
```

Comment:
I was though that StaticRoute would be a good builtin example how to use views. Or you didn't find this idea good as well?